target/linux/ramips/image/mt7621.mk

```diff
@@ -114,6 +114,33 @@ define Device/alfa-network_quad-e4g
 endef
 TARGET_DEVICES += alfa-network_quad-e4g
 
+define Device/ampedwireless_ally
```

Comment:
Consider naming this Device/ampedwireless_ally_common so it's obvious it's not a device itself.

Reply:
good idea; done.